src/panels/lovelace/cards/hui-logbook-card.ts

```diff
@@ -248,20 +256,38 @@ export class HuiLogbookCard extends LitElement implements LovelaceCard {
     this._lastLogbookDate = now;
   }
 
-  private _fetchPersonNames() {
-    if (!this.hass) {
-      return;
+  private async _fetchUserNames(hass: HomeAssistant) {
+    const userIdToName = {};
+
+    // Start loading all the data
+    const personProm = fetchPersons(hass);
+    const userProm = hass.user!.is_admin && fetchUsers(hass);
+
+    // Process persons
+    const persons = await personProm;
+
+    for (const person of persons.storage) {
+      if (person.user_id) {
+        userIdToName[person.user_id] = person.name;
+      }
+    }
+    for (const person of persons.config) {
+      if (person.user_id) {
+        userIdToName[person.user_id] = person.name;
+      }
     }
 
-    Object.values(this.hass!.states).forEach((entity) => {
-      if (
-        entity.attributes.user_id &&
-        computeStateDomain(entity) === "person"
-      ) {
-        this._persons[entity.attributes.user_id] =
-          entity.attributes.friendly_name;
```

Comment:
@bramkragten I just copied what [logbook panel has been doing](https://github.com/home-assistant/frontend/blob/8a33d174d737052724f5cec49ef26c043f4603f9/src/panels/logbook/ha-panel-logbook.ts#L202-L217), which is calling API to get persons. I thought that's a good/okay practice.

I updated all 3 places (logbook card, panel and 'more-info' dialog) to use `hass.states` instead of calling API to get persons. (`users` are still fetched via API)